### PR TITLE
Fix dependencies handling

### PR DESCRIPTION
**Pull request by tenntenn**
* Error handling
* Catch result of dependencies

### CODE REVIEW

**Comment by tenntenn** [comment]
I'd like to adapt [tagpr](https://github.com/Songmu/tagpr) instead of release branch. I created a PR (#17) which add a tagpr workflow of GitHub Actions. Then I changed the base branch of this PR to main.

File: version.txt [hunk 1]
```diff
@@ -1 +1 @@
-v.0.3.2
+v0.3.3
```
Comment (sonatard) [inline version.txt:1]: すいません version.txt上げ忘れていて次v0.3.4でお願いします！
https://github.com/gqlgo/gqlanalysis/releases

**Review by sonatard — APPROVED** [review]
LGTM！

File: version.txt (hunk 1 above)
Comment (tenntenn) [reply version.txt:1]: OK!

**Pull request merged**